assets/script.js

```diff
@@ -82,4 +82,20 @@ if (document.location.pathname.indexOf('fullcpgrid') > -1) {
 // Current year of footer
 const currentYearSpan = document.getElementById('currentYear')
 const currentYear = new Date().getFullYear()
-currentYearSpan.innerText = currentYear
\ No newline at end of file
+currentYearSpan.innerText = currentYear
+
+// Search bar 
```

Comment:
Here I would add a selector to get the input from the html, and then add an event listener that fires our function. This replacing the `onKeyPress` that was in our html. Use the `input` event because it is more generic and reacts in mobile or when a user pastes text

```js
// Search bar
const input = document.getElementById('searchbar')
input.addEventListener('input', searchCard)
```